### ISSUE TITLE
Erro Restart training 

### ISSUE THREAD

**Issue by anacastropsico** [DeepLabCut/DeepLabCut#2053]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I'm try restart train starto to point (snapshot-210000), using this im my pose file : init_weights: /content/drive/My Drive/Teste_DLC/new_projet/dlc-models/iteration-0/new_projetOct24-trainset95shuffle1/train/snapshot-210000. 
But the train start to 0 in my file folder it shows snapshot files being created starting at 10000 when I would expect it to continue training from 210000. I'm using resnet50, and 'imgaug' to train. I've used this same code before on a network without an 'imgaug' and it worked normally. I'm using GoogleColab

My train code: 
deeplabcut.create_training_dataset (drive_path, augmenter_type='imgaug')
deeplabcut.train_network (drive_path, allow_growth=True, max_snapshots_to_keep=8, saveiters=10000, maxiters=500000)

### Operating System

Windows

### DeepLabCut version

os.environ ['DLClight'] = 'True'
os.environ ['Colab'] = 'True'

### DeepLabCut mode

single animal

### Device type

Google Colab

### Relevant log output

```shell
Dont erro is relatad, this line 'init_weights: /content/drive/My Drive/Teste_DLC/new_projet/dlc-models/iteration-0/new_projetOct24-trainset95shuffle1/train/snapshot-210000.'    not is show in execution, I think somehow it's not being read, I even checked to see if it's in the correct pose file.
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by jeylau** [comment]
Hey @anacastropsico, could you please tell us what version of DeepLabCut you use? It looks to me like something we addressed back in January with #1653. If you update DLC with `pip install --upgrade deeplabcut`, you should get the expected behavior.

**Comment by anacastropsico** [comment]
> Hey @anacastropsico, could you please tell us what version of DeepLabCut you use? It looks to me like something we addressed back in January with #1653. If you update DLC with `pip install --upgrade deeplabcut`, you should get the expected behavior.

I'm using colab google. For this I using this comand to install deeplabcut:

 !pip install deeplabcut
os.environ ['DLClight'] = 'True'
os.environ ['Colab'] = 'True'
%cd /content/drive/My Drive/DLC/AP

**Comment by anacastropsico** [comment]
> Hey @anacastropsico, could you please tell us what version of DeepLabCut you use? It looks to me like something we addressed back in January with #1653. If you update DLC with `pip install --upgrade deeplabcut`, you should get the expected behavior.

This pip install --upgrade deeplabcut is solution, nice!!
![image](https://user-images.githubusercontent.com/81208194/203374036-f0ae32cd-7f55-49ee-ac61-7b21459aa94f.png)

**Comment by jeylau** [comment]
Great!☺️
